src/irmin-pack/inode.ml

```diff
@@ -755,6 +755,208 @@ struct
       in
       Irmin.Type.map I.t ~pre_hash (fun v -> { find = niet; v }) (fun t -> t.v)
 
+    module type DATA_FORMAT = sig
+      type vt
+      type t
+
+      exception Misconstructed_Data of string
+
+      val of_int : ?label:string -> int -> t
+      val to_int : ?label:string -> t -> int
+      val of_string : ?label:string -> string -> t
+      val to_string : ?label:string -> t -> string
+      val of_hash : ?label:string -> hash -> t
+      val to_hash : ?label:string -> t -> hash
+      val of_lazy_hash : ?label:string -> hash lazy_t -> t
+      val to_lazy_hash : ?label:string -> t -> hash lazy_t
```

Comment:
```suggestion
      val of_lazy_hash : ?label:string -> hash Lazy.t -> t
      val to_lazy_hash : ?label:string -> t -> hash Lazy.t
```

Stdlib documentation [decrees](https://caml.inria.fr/pub/docs/manual-ocaml/libref/Stdlib.Lazy.html) that we shouldn't use `lazy_t`, though I'm not sure why.